Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -589,8 +589,8 @@ public void driveBalanceExecute( )
     {
       driveStop(true);
     }
-
```

Comment:
Not sure why the DataLog line that was commented out was changed. It (like all Datalog calls) should start with a prefix of the subsystem (which it did).